source/fundamentals/crud/write-operations/update.txt

```diff
@@ -0,0 +1,293 @@
+.. _csharp-update-guide:
+
+================
+Update Documents
+================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you'll learn how to edit existing documents or upsert new documents 
+in your MongoDB collections using update operations.
+
+Sample Data
+~~~~~~~~~~~
+
+The examples in this guide use the ``restaurants`` collection
+from the ``sample_restaurants`` database. The documents in this
+collection use the following ``Restaurant`` class as a model:
+
+.. literalinclude:: /includes/fundamentals/code-examples/crud/update/Restaurant.cs
+   :language: csharp
+   :dedent:
+   :start-after: start-model
+   :end-before: end-model
+
+This collection is from the :atlas:`sample datasets </sample-data>` provided
+by Atlas. See the :ref:`<csharp-quickstart>` to learn how to create a free MongoDB cluster
+and load this sample data.
+
+Update Operations
+-----------------
+
+You can perform update operations in MongoDB with the following methods:
+
+- ``UpdateOne()``, which updates *the first document* that matches the query filter.
```

Comment:
The ordering gets a little funny here because the very next section describes what a query filter is, and it feels more natural to do so in the context of the parameters rather than the context of the method itself. One option I see is to reword the sentence you called out to something like "search" instead of "query filter."  What do you think?

Reply:
maybe search criteria?